examples/next-js-netlify/Earthfile

```diff
@@ -0,0 +1,38 @@
+VERSION 0.6
+FROM --platform=linux/amd64 node:latest
+WORKDIR /app
+
+deps:
+    ARG auth_token
+    ARG site_id
+    # Set environment variables required for building and deploying to Netlify
+    ENV NETLIFY_AUTH_TOKEN=$auth_token
```

Comment:
This should be passed in via a secret. See the `RUN --secret` option. I suspect you only need this for deployment - so you would normally put this in the `RUN netlify deploy` step.

Explanation: https://docs.earthly.dev/best-practices#use-secret-not-args-to-pass-secrets-to-the-build.

Reply:
The Netlify CLI requires the auth token and site id for build as well as for deploy (weird, right?). Regardless, I'll update to pass it as a secret.

Reply:
Do we have any examples that use secrets in `+examples1` or `+examples2`? Both `+build` and `+deploy` will fail without having those secrets passed appropriately. That's why I didn't include it in `+examples2`.

Reply:
We don't... I see the dilemma. We'd have to maintain the Netlify account + secrets just for this integration test. Perhaps it's ok not to include in our CI suite in that case. Feels like too much of a maintenance burden.